redwood.toml

```diff
@@ -0,0 +1,7 @@
+[web]
+  port = 8910
+  apiProxyPath = "/.netlify/functions"
+[api]
+  port = 8911
+[browser]
+  open = true
```

Comment:
This file probably shouldn't be here?

Reply:
This file is required for outline.test.ts to pass, yet I'm unsure how to work around it as my changes should be affecting this part. Tried removing the file with https://github.com/redwoodjs/redwood/pull/1424/commits/f7f30ecb5ee32fd8082f871c00f603eb789bbf5d but tests keep failing because of it.